authzed/api/materialize/v0/watchpermissionsets.proto

```diff
@@ -186,3 +192,21 @@ message BreakingSchemaChange {
   // change_at is the revision at which a breaking schema event has happened.
   authzed.api.v1.ZedToken change_at = 1;
 }
+
+message DownloadPermissionSetsRequest {
+  // optional_at_revision is a specific revision to download; for now this will
+  // just validate that it matches the backing store if provided.
+  authzed.api.v1.ZedToken optional_at_revision = 1;
+}
+
+message File {
+  // name is the filename of the downloadable file
+  string name = 1;
```

Comment:
why do you need a `name`, isn't that part of the url?

Reply:
That all lives inside the files here. There's a manifest file that we ship and includes a list of which files to expect in the payload. 

I don't really mind changing this but this is the design we merged, I can open it back for discussion.